Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
If this gets committed, I'll update the wiki with some recommandations and examples on how to launch Monocle inside a docker container... guess they are other things to talk about (e.g. creating a docker network and run a db inside a docker with persistency on local filesystem) so better tackle all those stuff on the wiki for people who would like to start with Docker ;-)